kv/mdbx/kv_mdbx.go

```diff
@@ -124,6 +126,16 @@ func (opts MdbxOpts) Readonly() MdbxOpts {
 	return opts
 }
 
+func (opts MdbxOpts) SafeNoSync() MdbxOpts {
+	opts.flags = opts.flags | mdbx.SafeNoSync
```

Comment:
default flags have Durable already, need remove it.
also now possible change flags by `mdbx.NewMDBX(logger).Flags(func(f uint) uint {return f ^ mdbx2.Durable | mdbx2.SafeNoSync})`

I'm really interesting in your experiments with SyncPeriod - because I myself had no time to experiment with it.

Reply:
ah, nice, I didn't notice the "f" parameter